docs/basics/DeepRLTutorial/TutorialDeepRL.rst

```diff
@@ -0,0 +1,605 @@
+Deep Reinforecement Learning in rlberry
+=======================================
+
```

Comment:
Remark : if you want to have the names of the authors in a comment, in rst this is done like that:
```
..
  Authors: Riccardo, Alena, Hector.
```